bindings/SofaRuntime/src/SofaPython3/SofaRuntime/Module_SofaRuntime.cpp

```diff
@@ -124,7 +124,7 @@ PYBIND11_MODULE(SofaRuntime, m) {
     if( !SceneLoaderFactory::getInstance()->getEntryFileExtension("py3") )
     {
         SceneLoaderFactory::getInstance()->addEntry(new SceneLoaderPY3());
-        sofa::helper::BackTrace::autodump();
+//        sofa::helper::BackTrace::autodump();  // We don't want to catch SIGINT in the python interpreter
```

Comment:
Isn't there a way to me more specific on the signal that trigger the autodump ?

Reply:
We could be that would mean changing the autodump method in SOFA core, or discarding the method entirely and implementing it in SofaPython3. I don't know who else uses the autodump method in SOFA. if this was made exclusively for SofaPython modifying it is a good idea, probably even better to move the whole function out of SOFA and into SofaPython3...?